src/coreclr/nativeaot/Bootstrap/main.cpp

```diff
@@ -164,6 +166,9 @@ static int InitializeRuntime()
 
     InitializeModules(osModule, __modules_a, (int)((__modules_z - __modules_a)), (void **)&c_classlibFunctions, _countof(c_classlibFunctions));
 
+    // Populate the values needed for debugging
```

Comment:
Wondering if we could just call this from `RhInitialize`.

Reply:
Sure!